docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that sdk and QtCreator should be installed and environments are sourced already.
```

Comment:
I think consistency is important and would recommend always capitalizing the acronym "SDK."